File: leetcode/AIcoding/1.Gemini-2.5/go_solutions/925-long-pressed-name.go
func isLongPressedName(name string, typed string) bool {
	i := 0 // Pointer for name
	j := 0 // Pointer for typed

	for j < len(typed) {
		if i < len(name) && name[i] == typed[j] {
			// Character matches name, advance both pointers
			i

File: leetcode/AIcoding/5.Qwen-3/javascript_solutions/925-long-pressed-name.js
var isLongPressedName = function(name, typed) {
    let i = 0, j = 0;
    while (i < name.length || j < typed.length) {
        if (name[i] !== typed[j]) {
            if (j > 0 && typed[j] === typed[j - 1]) {
                j++;
            } else {
                return false;
            }
        } else {
            i++;
            j++;
        }
    }
    return true;
};